chain/types/message_receipt_cbor.go

```diff
@@ -0,0 +1,220 @@
+package types
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/filecoin-project/go-state-types/exitcode"
+	cbg "github.com/whyrusleeping/cbor-gen"
+	"golang.org/x/xerrors"
+)
+
+// This file contains custom CBOR serde logic to deal with the new versioned
+// MessageReceipt resulting from the introduction of actor events (FIP-0049).
+
+var lengthBufMessageReceipt = []byte{132}
```

Comment:
thats probably not correct, the header includes number of fields.

Reply:
Fixed.